Client/Mobile/src/styles/styles.tsx

```diff
@@ -294,6 +294,10 @@ const styles = StyleSheet.create({
         flexDirection: 'row', 
         justifyContent: 'flex-end', 
         alignItems: 'center' 
+    },
+    error:{
+        color:'red',
```

Comment:
Don't use static colors or hex values, create one const color in styles.tsx and use it.